src/test/components/ProfileViewerContainer.test.js

```diff
@@ -0,0 +1,60 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+// @flow
+import * as React from 'react';
+import { shallowWithStore } from '../fixtures/enzyme';
```

Comment:
This utility is useful for shallow tests so that the root element is actually our tested element, and not `<Provider>`. It may be possible to use `<Provider>` and proper use of `dive()` but I think this is less simple... But tell me what you think!

Reply:
This works well I think. Thanks for figuring this out.